plugins/GSdx/GSRendererHW.cpp

```diff
@@ -58,7 +58,11 @@ void GSRendererHW::SetScaling()
 	int fb_width = max({ (int)m_context->FRAME.FBW * 64, crtc_size.x , 512 });
 	// GS doesn't have a specific register for the FrameBuffer height. so we get the height
 	// from physical units of the display rectangle in case the game uses a heigher value of height.
-	int fb_height = (fb_width < 1024) ? max(512, crtc_size.y) : 1024;
+	//
+	// Gregory: the framebuffer must have enough room to draw
+	// * at least 2 frames such as FMV (see OI_BlitFMV)
+	// * high resolution game such as snowblind engine game
+	int fb_height = max(1024, 2*crtc_size.y)
```

Comment:
@turtleli 
Please don't follow my example ;)

@ssakash 
I tried to do something. It is far from ideal, it will lose a couple of frames after a pause. It would be so much better to fix the texture cache allocation crazyness. And then drop all this mess.